packages/babel-parser/src/tokenizer/index.js

```diff
@@ -1377,34 +1371,38 @@ export default class Tokenizer extends ParserErrors {
 
   // Reads template string tokens.
 
+  readTemplateMiddle(): void {
```

Comment:
This can return both a `templateMiddle` or `templateTail`? Maybe `readTemplateAfterSubsitution`, or `readTemplateContinuation`?

Reply:
I slightly prefer `readTemplateContinuation` because it is shorter. Will also rename `readTmplToken` to `readTemplateToken` for consistency.